tests/system/model_monitoring/test_app.py

```diff
@@ -241,7 +218,6 @@ def _test_v3io_records(
         last_request: typing.Optional[datetime] = None,
         error_count: typing.Optional[float] = None,
     ) -> None:
-        cls._test_parquet(ep_id, inputs, outputs)
```

Comment:
remove `inputs` and `outputs` params from func signature (and adjust the callers as well)